employees.rb

```diff
@@ -0,0 +1,159 @@
+class Employee
+  attr_accessor:full_name
+  attr_accessor:id
+
+  def initialize(full_name, id)
+    @full_name = full_name
+    @id = id
+  end
+
+  def surname
+    @full_name.split(' ', 2).last
+  end
+end
+
+class Programmer < Employee
+  attr_accessor:languages
+
+  def initialize(full_name, id, languages)
+    super(full_name, id)
+    @languages = languages
+  end
+end
+
+class OfficeManager < Employee
+  attr_accessor:office
+
+  def initialize(full_name, id, office)
+    super(full_name, id)
+    @office = office
+  end
+end
+
+
+
+
+def add_employee(employees)
+  puts '[Add an employee]'
+  print 'Full name: '
+  full_name = gets.chomp
+  print 'ID: '
+  id = gets.chomp
+  print "Is the person [e]mployee, [p]rogrammer or an [o]ffice menager? "
+  role = gets.chomp.downcase
+  case role
+  when role='e' then
```

Comment:
Did you check if this works? It seems to me that `role = 'e'` should change the value of `role` to `e` instead of checking for it's equality. Also, when you use a `case` you usually don't have to do additional checks in each `when`. 

E.g.
```ruby
case role
  when 'e' then bla
end
```

Reply:
I dont know how, but to me it looks like its working with original code. I will change it for better readability of code, thanks for the tip.